src/index.ts

```diff
@@ -392,10 +380,10 @@ export class GraphRequest {
      */
     private sendRequestAndRouteResponse(requestBuilder:request.SuperAgentRequest, callback?:GraphRequestCallback):Promise<any>|void {
         // return a promise when Promises are supported and no callback was provided
-        if (typeof callback === "undefined" && typeof Promise !== "undefined") {
+        if (callback == null && typeof Promise !== "undefined") {
             return this.routeResponseToPromise(requestBuilder);
         } else {
-            this.routeResponseToCallback(requestBuilder, callback);
+            this.routeResponseToCallback(requestBuilder, callback || function(){});
```

Comment:
Pass a dummy function in so we don't have to worry about whether it exists or not in routeResponseToCallback.